Neos.Fusion/Tests/Functional/FusionObjects/DataStructureTest.php

```diff
@@ -145,4 +146,35 @@ public function unsetUntypedChildKeysWillRenderAsDataStructure(): void
         $view->setFusionPath('dataStructure/unsetUntypedChildKeyWillRenderAsDataStructure');
         self::assertEquals(['buz' => 456, 'keyWithUnsetType' => ['bat' => 123]], $view->render());
     }
+
+    /**
+     * @test
+     */
+    public function unsetChildKeyWillNotRender(): void
+    {
+        $view = $this->buildView();
+        $view->setFusionPath('dataStructure/unsetChildKeyWillNotRender');
+        self::assertEquals(['foo' => 'bar'], $view->render());
+    }
+
+    /**
+     * @test
+     * NOTE: This test mainly documents the current behavior. "null1" is removed by the {@see PositionalArraySorter} currently
```

Comment:
This test mainly documents the current behavior:

The fusion code is:
```fusion
dataStructure.nulledChildKeyWillRenderAsNull = Neos.Fusion:DataStructure {
  foo = 'bar'
  null1 = null
  null2 = ${null}
}
```

And `null1` is removed by the `PositionalArraySorter` – This is a separate issue that could be resolved with #3350

Reply:
The issue you linked is incorrect i think?

Reply:
Yes, sorry, fixed